photutils/daogroup.py

```diff
@@ -0,0 +1,138 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""Functions for implementing the DAOGROUP algorithm proposed by Stetson 1987.
+"""
+
+from __future__ import division
+import numpy as np
+from astropy.table import Column, Table, vstack
+
+__all__ = ['daogroup']
+
+def daogroup(starlist, crit_separation):
+    """
+    This is an implementation which follows the DAO GROUP algorithm presented
+    by Stetson (1987).
+
+    GROUP divides an entire starlist into sets of distinct, self-contained
+    groups of mutually overlapping stars.
+
+    GROUP accepts as input a list of stars and their approximate
+    brightenesses relative to a model stellar intensity profile for the frame
+    and determines which stars are close enough to be capable of
+    adversely influencing each others' profile fits.
+
+    Parameters
+    ----------
+    starlist : `~astropy.table.Table` or array-like
+        List of stars positions.
+        If `~astropy.table.Table`, columns should be named 'x_0' and 'y_0'.
+        Additionally, 'flux_0' may also be provided. 
+        TODO: If array-like, it should be either (x_0, y_0) or
+        (x_0, y_0, flux_0).
+        If 'starlist' only contains x_0 and y_0, 'crit_separation' must be
+        provided.
```

Comment:
so, if `flux_0` is there, how does that make `crit_separation` not required? Are you assuming some fixed FWHM?